README.md

```diff
@@ -33,3 +21,50 @@ run `./gradlew addKtlintFormatGitPreCommitHook`. As the result, the `.git` folde
 - [x] [Multiple Operator Comparison](./docs/transformations/MultipleOperatorComparison.md)
 - [x] [Multiple Target Assignment](./docs/transformations/MultipleTargetAssignmentTransformation.md)
 - [x] [Outer Not Elimination](./docs/transformations/OuterNotElimination.md)
+
+## Installation
+
+Just clone the repo by `git clone https://github.com/JetBrains-Research/ast-transformations`.
+
+
+## Getting started as a tool
+
+Run the command `./gradlew :cli -Pinput=<Input directory with python files> -Poutput=<Output directory> -Pyaml=<YAML config path>`.
+The YAML configuration file example can be found [here](./config.yaml).
+Set of the transformations is applied `n transformations` times. Each round for each transformation the random  
+number between `0.0` and `1.0` is generated. If this value less than `p` for the current transformation,
+the transformation will be applied. The result for each round is stored in the separated folder.
+
+
+## Getting started as a library
+
+Import the plugin in your plugin:
```

Comment:
Sounds a little bit strange, maybe somehow rephrase it? For example, "You can use this plugin as a library in your plugin by importing it in `settings.gradle.kts` and `build.gradle.kts` files"